sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/services.py

```diff
@@ -15,7 +21,20 @@
 module_logger = logging.getLogger(__name__)
 
 
-class JobServiceSchema(PathAwareSchema):
+class JobServiceBaseSchema(PathAwareSchema):
+    port = fields.Int()
+    endpoint = fields.Str(dump_only=True)
+    status = fields.Str(dump_only=True)
+    nodes = fields.Str()
```

Comment:
What's an example of `nodes`? Shouldn't this be a list of strings?

Reply:
The services will either run in all or none as per the discussion with IJ team. Will double check with Srujan as mentioned in the other comment.